cookbook-front-end/src/Components/RecipeDetail.tsx

```diff
@@ -31,15 +32,22 @@ query Recipe($recipeDetailId: Int){
 }`;
 
 
-export interface Props {
-    recipeDetailId: number
-}
+// export interface Props {
+//     recipeDetailId: number
+// }
+
+const RecipeDetail = () => {
+    console.log("Hey we're in recipe detail");
+
+    let { recipeDetailId } = useParams();
```

Comment:
This is how you grab url params from the route.

Reply:
ooo nice. gotta love hooks.